TombEngine/Objects/Generic/Switches/cog_switch.cpp

```diff
@@ -60,7 +60,7 @@ namespace TEN::Entities::Switches
 
 		// Door was not found, do ordinary collision and exit.
 
-		if (door == nullptr)
+		if ((door == nullptr) && (switchItem->TriggerFlags == 0x00))
```

Comment:
You can just use `!switchItem->TriggerFlags`. No need to use `== 0x00`.

Reply:
Sure, I'm changing it.